src/pages/taskList/TaskListPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import {MultipleSelectChip, TaskList, Title} from '../components/Shared.ts';
+import {label} from "../../../utils/mockData.ts";
+import SearchText from "../components/filters/SearchText.tsx";
+import {useTaskListHook} from "./hook.ts";
+import {Divider, Grid} from "@mui/material";
+import {Structure} from "../../../components/Structure.tsx";
+
+export default function Index() {
```

Comment:
index.ts needs to be just a barrel, this is something from Next.js and here we are working in React vanilla

Reply:
create a taskListPageContainer.tsx and add this logic inside